libkbfs/disk_block_cache.go

```diff
@@ -52,14 +51,21 @@ type diskBlockCacheConfig interface {
 
 // DiskBlockCacheStandard is the standard implementation for DiskBlockCache.
 type DiskBlockCacheStandard struct {
-	config   diskBlockCacheConfig
-	maxBytes uint64
-	log      logger.Logger
-	// This protects the disk caches from being shutdown while they're being
-	// accessed.
+	config     diskBlockCacheConfig
+	log        logger.Logger
+	maxBytes   uint64
+	maxBlockID []byte
+	// Track the number of blocks in the cache per TLF and overall.
```

Comment:
This comment is a bit unclear. I guess it refers to `tlfCounts` and `numBlocks`? And then `tlfSizes` and `currBytes` have the same relationship where they track the total/per-TLF byte size? Maybe rewrite comment to clarify.